package.json

```diff
@@ -67,7 +67,7 @@
     "build:es": "babel src -d es --env-name=es",
     "build:node": "babel src -d .",
     "build": "npm run build:browser && npm run build:node && npm run build:es",
-    "pretest": "npm run lint && npm run build",
+    "pretest": "npm run build && npm run lint",
```

Comment:
I think this is one of the things making my previous builds to fail, I'll retry the PR I'd reverted.

Reply:
Exactly